### PR TITLE
ISSUE-39 Arina Ulanova

### PR DESCRIPTION
**Pull request by Silvery-Moon**
Проверь, пожалуйста)

### CODE REVIEW

File: tasks [hunk 1]
```diff
@@ -55,9 +55,9 @@
 75 + 40
 52 - 31
 36 - 27
-18 + 22
-62 - 19
-47 - 12
+18 + 22 = 40
+62 - 19 = 42
```
Comment (Reason239) [inline tasks]: Are you sure about this one?

Comment (Silvery-Moon) [reply tasks]: Oh i'm so inattentive

**Comment by Silvery-Moon** [comment]
Corrected. Is everything right now?

**Comment by Reason239** [comment]
Ok, looks great

**Pull request merged**